[R] Cannot install RServe:  R was configured without --enable-R-shlib or --enable-R-static-lib

## Description
Trying to install the Rserve component of R, getting the following error.

If you executed a specific command:
```shell
in R -
install.packages("Rserve")
```

Output:
```shell
.........
* installing *source* package ‘Rserve’ ...
** package ‘Rserve’ successfully unpacked and MD5 sums checked
checking whether to compile the server... yes
configure: error: R was configured without --enable-R-shlib or --enable-R-static-lib

*** Rserve requires R (shared or static) library.                       ***
*** Please install R library or compile R with either --enable-R-shlib  ***
*** or --enable-R-static-lib support                                    ***

 Alternatively use --without-server if you wish to build only Rserve client.


ERROR: configuration failed for package ‘Rserve’
* removing ‘/usr/local/lib64/R/library/Rserve’
```

## Versions and system information
chromebrew: 0.4.3

ChromeOS:
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4613139
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=76
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R64-10176.76.0
CHROMEOS_RELEASE_BUILD_NUMBER=10176
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=64
CHROMEOS_RELEASE_DESCRIPTION=10176.76.0 (Official Build) stable-channel eve 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=stable-channel
CHROMEOS_RELEASE_VERSION=10176.76.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10176.76.0

Model: Pixelbook

Architecture: x86_64


What I think needs to be done:
- Recompile package with options


---

--- a/packages/r.rb
+++ b/packages/r.rb
@@ -3,21 +3,13 @@ require 'package'
 class R < Package
   description 'R is a free software environment for statistical computing and graphics.'
   homepage 'https://www.r-project.org/'
-  version '3.4.3-1'
-  source_url 'https://cran.r-project.org/src/base/R-3/R-3.4.3.tar.gz'
-  source_sha256 '7a3cb831de5b4151e1f890113ed207527b7d4b16df9ec6b35e0964170007f426'
+  version '3.5.1'
+  source_url 'https://cran.r-project.org/src/base/R-3/R-3.5.1.tar.gz'
+  source_sha256 '0463bff5eea0f3d93fa071f79c18d0993878fd4f2e18ae6cf22c1639d11457ed'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/r-3.4.3-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/r-3.4.3-1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/r-3.4.3-1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/r-3.4.3-1-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'cfbe1876271be723e1477c325830917fa45acb5e36187bbcb3d8ce835668357e',
-     armv7l: 'cfbe1876271be723e1477c325830917fa45acb5e36187bbcb3d8ce835668357e',
-       i686: '7d69683c6b3c9ce41718652eccc1b4ddb4cba42fee7060c00a06d3afdf25ebca',
-     x86_64: 'b70dbd51af3b9c1676bc4e9c3d90ff4169556a6937f81a51252c7be81b365f05',
   })
 
   # depends_on 'gfortran'       # require gfortran enabled gcc
@@ -27,14 +19,15 @@ class R < Package
   depends_on 'bz2'
   depends_on 'curl'
   depends_on 'openssl'
-  depends_on 'readline'
+  depends_on 'readline7'
   depends_on 'icu4c'
 
   def self.build
     system './configure',
       "--prefix=#{CREW_PREFIX}",
       "--libdir=#{CREW_LIB_PREFIX}",
-      '--with-x=no' # X is not available
+      '--with-x=no', # X is not available
+	  '--enable-R-shlib'
     system 'make'
   end
 

